gensrc/thrift/Descriptors.thrift

```diff
@@ -102,8 +102,9 @@ enum TSchemaTableType {
     SCH_USER_PRIVILEGES,
     SCH_VARIABLES,
     SCH_VIEWS,
-    SCH_INVALID,
-    SCH_ROWSETS
+    SCH_ROWSETS,
+    SCH_BACKENDS,
+    SCH_INVALID
```

Comment:
do not change the order of enums. 
it will core during rolliing upgrade be because be depend on order or id to check enum value.

Reply:
done